CalendarKit-Swift/SwiftCal/ICSEventParser.swift

```diff
@@ -202,6 +203,15 @@ class ICSEventParser: NSObject {
 
         let eventScanner = Scanner(string: icsString)
         eventScanner.scanUpTo(ICS.summary, into: nil)
+        
+        if eventScanner.isAtEnd {
+            // Handle `SUMMARY;LANGUAGE=en-US:Event title` format
+            eventScanner.scanLocation = 0
+            eventScanner.scanUpTo(ICS.summary2, into: nil)
+            eventScanner.scanUpTo(":", into: nil)
```

Comment:
Are colons allowed in the title?

Reply:
I think so, but it shouldn't be a problem because this will simply scan up to the first ":", then read the full title afterward.

Reply:
I updated the test to validate that: https://github.com/superhuman/SwiftCal/pull/15/commits/58b12e6393a87c29deddc7cc9369a7f465062bc7